recipes/tldr/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "tldr" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 731b10bf38bb84a73560408a783b26d08ac52eab2e9a55f89dba9a3c23a72892
+
+build:
+  number: 0
+  entry_points:
+    - tldr = tldr:cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - argcomplete
+    - colorama
+    - pip
+    - python
+    - termcolor
+  run:
+    - argcomplete
+    - colorama
+    - python
+    - termcolor
+
+test:
+  commands:
+    - tldr --help
+  requires:
+    - pytest
+    - pytest-runner
```

Comment:
I don't think these are actually needed to run the command you're using as a test?
```suggestion
```